MSAL/src/MSALResult.m

```diff
@@ -55,15 +61,18 @@ + (MSALResult *)resultWithAccessToken:(NSString *)accessToken
     return result;
 }
 
-+ (MSALResult *)resultWithAccessTokenItem:(MSALAccessTokenCacheItem *)cacheItem
++ (MSALResult *)resultWithAccessToken:(MSIDAccessToken *)accessToken
 {
-    return [self resultWithAccessToken:cacheItem.accessToken
-                             expiresOn:cacheItem.expiresOn
-                              tenantId:cacheItem.tenantId
-                                  user:cacheItem.user
-                               idToken:cacheItem.rawIdToken
-                              uniqueId:cacheItem.uniqueId
-                                scopes:[cacheItem.scope array]];
+    __auto_type idToken = [[MSIDAADV2IdTokenWrapper alloc] initWithRawIdToken:accessToken.idToken];
```

Comment:
nit: if we're passing id token to result as raw ID token, maybe it's better to pass raw id token to user as well

Reply:
raw idToken is stored in MSIDAccessToken. My opinion it is better to use classes instead of 'raw' values...